resources/js/features/root/pages/AdminFacilitiesList.page.tsx

```diff
@@ -30,11 +29,7 @@ export default (() => {
           },
           url: {
             columnDef: {
-              cell: (c) => {
-                const href = () => `/${c.getValue()}`;
-                // TODO: The link can be inaccessible for the current user, handle this better.
-                return <A href={href()}>{href()}</A>;
-              },
+              cell: (c) => `/${c.getValue()}`,
```

Comment:
Why is this change valid? will it be automatically converted to the link?

Reply:
Nie, to nie powinny być linki, bo w typowej sytuacji globalny admin nie ma dostępu do URLi wszystkich placówek. Być może ma dostęp do niektórych, ale raczej nie ma sensu wyświetlać żadnych działających linków w tym miejscu.

Reply:
W pewnym sensie to rozwiązuje to TODO.